src/registry.js

```diff
@@ -276,7 +276,7 @@ function registerTraceModule(_module) {
     // add `PlotlyGeoAssets` global to stash references to all fetched
     // topojson / geojson data
     if((bpmName === 'geo' || bpmName === 'mapbox') &&
-        (typeof window !== undefined && window.PlotlyGeoAssets === undefined)
+        (typeof window !== 'undefined' && window.PlotlyGeoAssets === undefined)
```

Comment:
The condition here is still not strict enough.
I suggest we use:
```suggestion
        (typeof window === 'object' && window.PlotlyGeoAssets === undefined)
```

Reply:
That's the only place we do `typeof window` at all, everywhere else we just assume it's present. Given that as is it's always going to give `true`, can't we just remove that condition?

Reply:
We also test for `window` here: 
https://github.com/plotly/plotly.js/blob/73e05ce7cd1f7174877ef90ac98d988eed115e50/src/lib/is_plain_object.js#L9